README.md

```diff
@@ -50,7 +50,7 @@ Options:
   -v, --verbose                   Use multiple times to increase verbosity
   --config-file PATH              Config file to use. Use 'no' to disable
                                   config file. Default value: first of
-                                  ./.vault.yml, ~/.vault.yml, /etc/vault.yml
+                                  ./vault.yml, ~/.vault.yml, /etc/vault.yml
```

Comment:
I thought you wanted to switch to `~/.config/vault.cli` too ? Can be done in another PR though.